pkg/components/imguploader/gcs/gcsuploader.go

```diff
@@ -149,7 +149,7 @@ func (u *Uploader) uploadFile(
 	key string,
 ) error {
 	u.log.Debug("Opening image file", "path", imageDiskPath)
-	fileReader, err := os.Open(imageDiskPath)
+	fileReader, err := os.Open(filepath.Clean(imageDiskPath))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.